index.html

```diff
@@ -7,29 +7,39 @@
     <title> Lynn's portofolio</title>
     <link rel="stylesheet" type="text/css" href="style.css">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css">
+
 </head>
 <body>
+  <div id="time">
+    <h1>9:41</h1>
+</div>
+<i class="fas fa-network-wired common-icon"></i> 
+<i class="fas fa-wifi common-icon"></i>
+<i class="fas fa-battery-full common-icon"></i>
     <header>
         <h2>John Doe</h2>
 
         <nav>
           <ul class="menu">
-            <li><a href="#">Home</a></li>
-            <li><a href="#">About</a></li>
-            <li><a href="#">Contacts</a></li>
+            <li><a href="#"><i class="fas fa-bars"></i></a></li>
           </ul>
```

Comment:
Replace the ul tag with div, this is not a list so no need to use ul, just use div and a tags